dts/arm/st/n6/stm32n6.dtsi

```diff
@@ -640,6 +640,19 @@
 			dma-offset = <0>;
 			status = "disabled";
 		};
+
+		xspi2: spi@5802a000 {
+			compatible = "st,stm32-xspi";
+			reg = <0x5802A000 0x1000>;
+			interrupts = <171 0>;
+			clock-names = "xspix", "xspi-ker", "xspi-mgr";
+			clocks = <&rcc STM32_CLOCK(AHB5, 12)>,
+				 <&rcc STM32_SRC_CKPER XSPI2_SEL(1)>,
+				 <&rcc STM32_CLOCK(AHB5, 13)>;
```

Comment:
It seems we miss drivers hooks to handle these clocks, though I remind I've seen them in a previous review. Are you missing a commit in this PR ?

Reply:
What do you mean by drivers hooks? In the XSPI Flash driver, these clocks are currently handled by their index rather than their name. I haven't changed the behavior. Should I update the driver to use the clock names (like for XSPI PSRAM)?

Reply:
I think I've been confused with PSRAM driver. I'll have a look.

Reply:
Ok, we can work with that as it is already working for other series, but we may want to make it more robust in the future.